### ISSUE TITLE
Cursor is hidden without being moved

### ISSUE THREAD

**Issue by devnoname120** [Airblader/unclutter-xfixes#23]
I use unclutter with a tiling window manager (i3) since most of the time I don't use the mouse.
However, the hidden cursor still interacts with the underlying elements, and this is particularly annoying when browsing the web because it will trigger the `onmouseover` event of all elements it goes above.

Would it be possible to add an option so that the cursor would be moved at a chosen location when it's hidden? It would come back at the previous location when unhiding the mouse by moving it.

Thank you for this awesome tool.

**Comment by Airblader** [comment]
Sure, PRs welcome. Note that it'll have to find a good way to deal with things like changing screen configurations in the meantime and that it should of course be an additional option, not a default behavior.

**Comment by devnoname120** [comment]
I agree. I don't think I will do it since I don't have good knowledge about how these things are handled. I would rather spend my time on other opensource projects I maintain rather than spending significant time learning what I need to know to implement it.

**Comment by amosbird** [comment]
Moving cursors would result in unexpected side effects, for instance https://github.com/jcs/xbanish/issues/40

I wonder if it's possible to make the X window ignore mouse hover event when the cursor is invisible?

**Comment by daniel-Jones** [comment]
this began annoying me, to quickly and hackily get the pointer off screen, top left i added to cursor_hide() in cursor.c
``
XWarpPointer(display, None, None, 0,0,0,0, -1920, -1080);
``
all previous discussed problems are a thing but it works good enough for me.

**Comment by mjkalyan** [comment]
> Moving cursors would result in unexpected side effects, for instance [jcs/xbanish#40](https://github.com/jcs/xbanish/issues/40)
> 
> I wonder if it's possible to make the X window ignore mouse hover event when the cursor is invisible?

If possible, making X ignore some user-defined set of mouse events while hidden would be the most elegant solution in my opinion. If I think of a way to do that (or someone with better knowledge of X gives me some direction) I could submit a PR.

**Comment by amosbird** [comment]
> I could submit a PR.

@mjkalyan fantastic.

**Comment by pc00per** [comment]
Can there be any way to instead of hiding cursor, but to completely disable  ? That would fix this issue btw.
